python/lsst/ts/standardscripts/base_script_test_case.py

```diff
@@ -61,6 +63,78 @@ async def test_something(self):
 
     _index_iter = utils.index_generator()
 
+    _broker_configuration = None
+    _schema_registry_url = None
+
+    async def asyncTearDown(self) -> None:
+        """Runs when the test is done.
+
+        This will delete all the topics and schema from the
+        kafka cluster.
+        """
+        if self._broker_configuration is None:
+            async with salobj.Domain() as d, salobj.SalInfo(d, "Test", 0) as salinfo:
```

Comment:
Is the `Test` CSC just here as a placeholder since the constructor needs it?

Reply:
yes..